x-pack/plugins/cases/server/client/cases/update_custom_field.test.ts

```diff
@@ -0,0 +1,227 @@
+/*
```

Comment:
Since we will stop sending the type could you please add a test where the type of the value sent does not match the type of the custom field in the configuration?

Setting a `TOGGLE` to `'foobar'`, for example.